src/wallet/wallet.cpp

```diff
@@ -7731,23 +7731,82 @@ std::optional<libzcash::UnifiedAddress> UnifiedAddressForReceiver::operator()(co
     return std::nullopt;
 }
 
+PrivacyPolicy PrivacyPolicyMeet(PrivacyPolicy a, PrivacyPolicy b)
```

Comment:
A meet m = a∧b is defined as greatest lower bound of a and b (i.e. m ≤ a, m ≤ b, and w ≤ m for any other lower bound w ≤ a, w ≤ b). So we infer from the implementation of this function that x ≤ y means "Policy x allows everything that policy y allows."

```suggestion
/// Let `(P, ≤)` be the partially ordered set of policies such that `x ≤ y` means
/// "Policy `x` allows everything that policy `y` allows."
///
/// This function returns the meet (greatest lower bound) of `a` and `b`, i.e.
/// the strictest policy that allows everything allowed by `a` and also everything
/// allowed by `b`.
PrivacyPolicy PrivacyPolicyMeet(PrivacyPolicy a, PrivacyPolicy b)
```

Reply:
I checked that the graph at #5693, the implementations of `TransactionStrategy::{AllowRevealedAmounts, AllowRevealedRecipients, AllowRevealedSenders, AllowLinkingAccountAddresses}`, and the implementation of this function are all consistent.

Reply:
I’ll integrate this into the comment in the .h.

Do we prefer `///` over `/**`? I don’t have a preference, just want to match styles.

Reply:
We seem to prefer `/**`:
```shell
$ zcutil/distclean.sh
$ grep --include='*.cpp' --include='*.h' -IRn -A20 '^/[*][*][^*]' |grep '[-] [*] ' |wc --lines
1160
$ grep --include='*.cpp' --include='*.h' -IRn -A20 '^/// ' |grep ':///' |wc --lines
375
```